package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "description": "Helping to connect people to appropriate services, meeting their time, location and accessibility needs.",
   "scripts": {
-    "brunch-build": "yarn brunch build",
+    "brunch-build": "node headerItems.js & yarn brunch build",
```

Comment:
This should be a double ampersand here as it means it must run and be successful.

Reply:
Done